harmonisation/specimen/biostratigraphicTerms.md

```diff
@@ -0,0 +1,28 @@
+# Biostratigraphic terms
+
+This is a set of terms which are combined in this single file.
+This is mainly te reduce the amount of files.
```

Comment:
This is **to** reduce the amount of files

Reply:
Reduce the files in the backend?

Reply:
Will change